htdocs/manage/circle/editfilters.bml

```diff
@@ -87,7 +87,9 @@ _c?>
             next unless $post_key =~ /^editfriend_(groupmask|maskhi)_(\w+)$/;
 
             my $trusted_u = LJ::load_user( $2 );
-            next unless $trusted_u;
+            # User might have been removed from circle between load and
+            # submit; don't re-add.
+            next unless $trusted_u && $u->trusts($trusted_u);
```

Comment:
Nit: style guide says to use space inside the parenthesis: `$u->trusts( $trusted_u );`